zstacklib/zstacklib/utils/ovn.py

```
@@ -376,26 +382,28 @@ def getVnicsByVmUuid(self, vmUuid, brName="br-int"):
                     cmd=CtlBin, brName=brName, nicName=nicName, nicType=nicType, srcPath=srcPath, nicUuid=nicUuid,
-                    vmUuid=vmUuid)
+                    vmUuid=vmUuid, interfaceId=interfaceId)
             else:
                 cmd = CtlBin + '--may-exist add-port {brName} {nicName} ' \
                                '-- set Interface {nicName} type={nicType} options:vhost-server-path={srcPath} ' \
-                               '-- set interface {nicName} external-ids:iface-id={nicName}_{nicUuid} ' \
+                               '-- set interface {nicName} external-ids:iface-id={interfaceId} ' \
                                '-- set interface {nicName} external-ids:vm-uuid={vmUuid}'.format(
                     brName=brName, nicName=nicName, nicType=nicType, srcPath=srcPath, nicUuid=nicUuid,
-                    vmUuid=vmUuid)
+                    vmUuid=vmUuid, interfaceId=interfaceId)
```

Ruff (0.15.12)

[error] `.format` call has unused named argument(s): nicUuid

Remove extra named arguments: nicUuid

(F522)